### DISCUSSION TITLE
Case Sensitive options for String Like operator

### DISCUSSION THREAD

**Discussion by thutch** [cagrin/LikeComparison#13] [Q&A]
Is it possible to set the Like comparison to be Case Sensitive?  
Example

"TEST".Like("test")  
for Case sensitive, this should return a false.

**Comment by cagrin** [comment]
Yes, sure. You can use PostgreSql variant, which is case-sensitive:

```using LikeComparison.PostgreSql;```
...
```bool isLike = "TEST".Like("test"); // returns false```
